extensions/wikia/Email/Controller/WatchedPageController.class.php

```diff
@@ -310,31 +337,4 @@ protected function getArticleLinkText() {
 	protected function getAllChangesText( $title ) {
 		return parent::getAllChangesText( $this->newTitle );
 	}
-
-	protected static function getEmailSpecificFormFields() {
```

Comment:
This need to get moved into the abstract class since it's code needed by all child classes.